.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+coverage/
```

Comment:
Hmm, I'm not entirely sure to be honest but it feels a bit like bloat to me since anyone can run a command and see it if they want to, is it common to leave this in?

Reply:
Im not sure to be honest, its a bit nit picky for me to ask. Its not doing any harm by reducing bloat so it could be left in without issue really